naresh/src/main/java/com/lftechnology/phpjava/ems/utlis/Router.java

```diff
@@ -0,0 +1,112 @@
+package com.lftechnology.phpjava.ems.utlis;
+
+import com.lftechnology.phpjava.ems.constants.Constant;
+import com.lftechnology.phpjava.ems.controllers.EmployeeController;
+import com.lftechnology.phpjava.ems.controllers.LoginController;
+import com.lftechnology.phpjava.ems.services.EmployeeService;
+import com.lftechnology.phpjava.ems.views.CommonViewUtility;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * RouterService
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 09 2016
+ */
+public class Router {
+
+    private static LoginController loginController = new LoginController();
+    private static EmployeeController employeeController = new EmployeeController();
+    private static CommonUtility commonUtility = new CommonUtility();
+
+    static {
+        try {
+            onRouteBootstrap();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Method to be executed on router startup.
+     * Will invoke checkAndCreateAdminUser() method
+     * to create admin user if the system doesn't have one.
+     *
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    public static void onRouteBootstrap() throws Exception {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS00112)